db/migrate/20180704190257_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      ## Database authenticatable
+      t.string :Genre, null: false, default: ''
```

Comment:
By convention, we let the column names of the database in lowcase.
We can make another migration renaming the columns.
Look for `rename_column` here: http://guides.rubyonrails.org/active_record_migrations.html